web/styles/dark_theme.css

```diff
@@ -1554,70 +1554,6 @@
             }
         }
     }
-
-    #mark_as_read_turned_off_banner {
```

Comment:
I would also recommend putting this code deletion in `unread_banner: Refactor HTML to use main view banner stylings`, since that's where the CSS stops being used.